### PR TITLE
#6 Even Fibonacci numbers

### PR DESCRIPTION
**Pull request by krishnatimilsina**
Please review the code.

### CODE REVIEW

File: src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.lftechnology.java.training.krishna.basics.even_fibonacci;
```
Comment (adhpawal) [inline src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java:1]: `_` feri dai ... :8ball:

File: src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.java.training.krishna.basics.even_fibonacci;
+
+import java.util.logging.Logger;
+
+
+public class EvenFibonacciDemo {
```
Comment (utsabneupane) [inline src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java:6]: use class level documentation

File: src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.java.training.krishna.basics.even_fibonacci;
+
+import java.util.logging.Logger;
+
+
```
Comment (sanjayapandey) [inline src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java:5]: Class level Javadoc is missing.

File: src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java (hunk 2 above)
Comment (krishnatimilsina) [inline src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java:6]: @utsabneupane Please visit the pull request: Largest palindrome product

File: src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java (hunk 1 above)
Comment (krishnatimilsina) [inline src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java:1]: @adhpawal Please visit the pull request: Largest palindrome product

File: src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java (hunk 3 above)
Comment (krishnatimilsina) [inline src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacciDemo.java:5]: @sanjayapandey Please visit the pull request: Largest palindrome product

File: src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java (hunk 1 above)
Comment (adhpawal) [inline src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java:1]: Sure

**Pull request closed without merging**